src/data/markdown/docs/01 guides/02 Using k6/14 Scenarios/03 Graceful stop.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 'Graceful stop'
+excerpt: ''
+---
+
+> ### 🎉 New in v0.27.0
+>
+> This feature is new as of version 0.27.0. Usage of this feature is optional and for the vast majority,
+> existing scripts and configurations will continue to work as before. For a list of breaking changes,
+> see the [k6 v0.27.0 release notes](https://github.com/loadimpact/k6/releases/tag/v0.27.0)).
+
+## Description
+
+This option is available for all executors except `externally-controlled` and allows the
+user to specify a duration to wait before forcefully interrupting them. The default value
+of this property is `30s`.
+
+## Example
+
+<div class="code-group" data-props='{"labels": [ "graceful-stop.js" ], "lineNumbers": "[true]"}'>
+
+```js
+import http from 'k6/http';
+
+export let options = {
```

Comment:
I had to read twice to understand how it works. I think we can simplify the example.

1 -  Use `duration` https://k6.io/docs/using-k6/options#duration instead of  `stages`. 

2 - Can we use an executor that it does not need to use `gracefulRampDown`.

Reply:
Yes, fixed. 👍